src/app/public/index.html

```diff
@@ -32,6 +32,15 @@
     <title>Open Apparel Registry (beta)</title>
     <!-- Environment Variables -->
     <script src="%PUBLIC_URL%/web/environment.js"></script>
+    <!-- TODO: Replace with actual Google Tag Manager snippet -->
+    <script>
+      if (window.ENVIRONMENT &&
+          window.ENVIRONMENT.REACT_APP_GOOGLE_ANALYTICS_KEY &&
+          window.ENVIRONMENT.REACT_APP_GOOGLE_ANALYTICS_KEY !== "" &&
+          window.ENVIRONMENT.ENVIRONMENT !== 'development') {
```

Comment:
This is here to demo how to implement this in production, but this line should be commented out to see the logging in development.